projects/isc-projects/dhcp/Makefile

```diff
@@ -0,0 +1,62 @@
+BASE_DIRECTORY:=$(abspath ../../../)
+GIT_TAG=$(shell cat GIT_TAG)
+
+REPO=dhcp
+REPO_OWNER=isc-projects
+
+SIMPLE_CREATE_BINARIES=false
+
+# this is a native build which we can not cross compile
+# ensure we are building to a directory based on the current
+# host platform to avoid ever creating a different arch'd
+# binary in the wrong folder
+BINARY_PLATFORMS?=linux/$(BUILDER_PLATFORM_ARCH)
+BINARY_TARGETS=$(foreach platform,$(BINARY_PLATFORMS),$(OUTPUT_BIN_DIR)/$(subst /,-,$(platform))/dhcp)
+
+HAS_LICENSES=false
+
+TAR_FILE_PREFIX=dhcp
+HAS_S3_ARTIFACTS=true
+SKIP_CHECKSUM_VALIDATION=true
+
+IMAGE_NAMES=
+
+EXCLUDE_FROM_CHECKSUMS_BUILDSPEC=true
+EXCLUDE_FROM_UPGRADE_BUILDSPEC=true
+
+BUILDSPEC_VARS_KEYS=BINARY_PLATFORMS
+BUILDSPEC_VARS_VALUES=BINARY_PLATFORMS
+BUILDSPEC_ARCH_TYPES=LINUX_CONTAINER ARM_CONTAINER
+
+# Since we build the arm and amd binaries on difference instances in codebuild
+# we do not want to delete missing files when s3 sync-ing from local to the bucket
+# since this would clobber the other build's artifacts
+UPLOAD_DO_NOT_DELETE=true
+
+
+include $(BASE_DIRECTORY)/Common.mk
+
+
+tarballs: gather-non-golang-licenses $(OUTPUT_DIR)/ATTRIBUTION.txt
+
+$(OUTPUT_BIN_DIR)/%: MAKEFLAGS=
```

Comment:
add this, i am doing something similar with ipxe and ran into this on my linux host

# since we are yum installing, we need to run the docker container as root
$(OUTPUT_BIN_DIR)/%: export USER=

Reply:
actually ignore that for now. i have a better idea, ill update this spot later.

side note: you likely wouldnt see this if you were building on a mac just because of the way users are handled in the builder base docker container.